content/_data/upgrades/2-277-1.adoc

```diff
@@ -0,0 +1,61 @@
+==== Configuration Form Modernization
+
+The transition from HTML `table` layout to HTML `div` layout in configuration forms requires changes in some plugins for compatibility.
+Jenkins administrators should update the installed plugins when they upgrade to Jenkins 2.277.1.
+Some older plugin releases will cause the Jenkins "Configure System" page to fail to save.
+Other older plugin releases will report Java runtime exceptions until they are updated to the most recent release.
+
+Popular plugins that need to be updated after upgrading to Jenkins 2.277.1 include:
+
+* plugin:branch-api[Branch API plugin]
+* plugin:envinject[Environment injector plugin]
+* plugin:favorite[Favorite plugin]
+* plugin:github-branch-source[GitHub branch source plugin]
+* plugin:ldap[LDAP plugin]
+* plugin:maven-plugin[Maven integration plugin]
+* plugin:parameterized-trigger[Parameterized trigger plugin]
+* plugin:token-macro[Token macro plugin]
+
+Use the Plugin Manager to update plugins immediately after upgrading to Jenkins 2.277.1.
+Make sure to _Check for Updates_ to ensure no outdated cached data hides updates.
+
+NOTE: Plugin maintainers should refer to the link:/doc/developer/views/table-to-div-migration/[table to div layout migration guide].
+The link:https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=20741[table to div dashboard] provides the status of issues related to this user interface improvement.
+
+==== Spring Security update
+
+Jenkins 2.277.1 has replaced the outdated Acegi Security library with the Spring Security library.
+The most popular plugins have already been updated as necessary to support the change.
+Refer to the link:https://github.com/jenkinsci/jep/blob/master/jep/227/compatibility.adoc[Spring Security compatibility table] for questions related to specific plugins.
+
+Privately maintained plugins may need changes to adapt to the change from Acegi Security to Spring Security.
```

Comment:
Applies to the change above as well, with higher likelyhood of a change

Reply:
Added as part of 0cb78e6d33